src/main/java/github/grace000/erversay/Constants/Headers.java

```diff
@@ -0,0 +1,6 @@
+package github.grace000.erversay.Constants;
+
+public final class Headers {
+    public final static String PIGGLY_BODY = "piggly wiggly";
```

Comment:
Does this really belong in the `Constants/Headers` file? It doesn't seem like a header...